src/global_data/4C_global_data_read.cpp

```diff
@@ -1521,6 +1552,9 @@ void Global::read_conditions(
   std::map<int, std::vector<int>> node_sets;
   get_node_sets_from_mesh(node_sets, mesh_reader);
 
+  std::map<int, std::vector<int>> element_block_nodes;
+  get_element_block_nodes_from_mesh(element_block_nodes, mesh_reader);
```

Comment:
Reminder to myself: this might be problematic for very large meshes. We already have an open point to evaluate the performance for larger examples in #653.